### ISSUE TITLE
Interactive Download Crops>Production Download As ...

### ISSUE THREAD

**Issue by UlrikeWood-Sichra** [FAOSTAT/faostat-ui#105]
I have selected my country, crop, item, years and click on DOWNLOAD AS, but nothing happens.

**Comment by Kalimaha** [comment]
right now it is necessary to preview the table in order to format or download it. this will be fixed jointly with issue #88

**Comment by Kalimaha** [comment]
Note for self: use https://github.com/FENIX-Platform/fenix-ui-olap/blob/master/tests/main.js#L42

**Comment by UlrikeWood-Sichra** [comment]
OK, now it worked, download after preview. But, it did neither preview nor download everything.
I selected for 2 countries area and production of cereals (list) and fruit (total). Production was completely missing for one country.
Additionally, the csv format I requested has a funny character at the very beginning, which creates a strange field name when imported into Foxpro with the wizzard. See 
![000csv_format_wrong](https://cloud.githubusercontent.com/assets/14282831/9885005/ef9dcde8-5bb0-11e5-80d2-13717acea5ff.JPG).
The generated field names are also not OK, because some of them start with a digit, which FoxPro does not want, and duplicate field names are also modified. Why not name them distinctly, eg for the values: Y2010, Y2011, ... and for the flags F2010, F2011, ...
The field names in FoxPro look like this now:
![000foxpro_fieldname](https://cloud.githubusercontent.com/assets/14282831/9885072/480c7204-5bb1-11e5-8779-c329b97d963b.JPG)
Besides, there is an additional column which was not in the csv file. And the text at the very end of the data is also not helpful since it creates noise in the imported file.
